specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/examples/integrations/GetIntegration_example.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "api-version": "2023-07-01-preview",
+    "scope": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "integrationId": "ad9a8e26-29d9-4829-bb30-e597a58cdbb8"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
doesn't integrations have 'groups' and 'users' properties (arrays of resources)?

Reply:
No, It's not part of the integraiton model